QUEST.tsv

```diff
@@ -2452,116 +2452,116 @@ QUEST_20151001_002459	The request can be accepted by the party leader.
-QUEST_20151001_002524	Even so, the Goddesses weren't there.{nl}The Goddesses who we trusted disappeared without a trace and no one prayed.{nl}People became mad and sad, and later only resignations were left behind.{nl}We started to prevent the chaos as much as we could, but as we did so, more misunderstandings were created. {nl}The deaths were always near and we became gradually emotionless. {nl}
-QUEST_20151001_002525	We are becoming emotionless to all the sadness and the pains.{nl}Like a dead man, we give up on everything and try not to do anything.{nl}As if we are seeing the end of the history of the kingdom...{nl}We are just keep repeating the names of the Goddesses.{nl}
-QUEST_20151001_002526	I will end my writing here and leave the kingdom.{nl}I don't like my feeling of helplessness and the depressions.{nl}I am going to leave to the place where I can dream for hope again...
+QUEST_20151001_002522	A gigantic tree rose up from deep underground the capital city.{nl}The size of the tree was big enough that it couldn't even be compared against any mountain.{nl}As a result, many cities were destroyed and many people lost their beloved ones.{nl}
+QUEST_20151001_002523	But, the disaster did not end there.{nl}The evil energy started to surround the grounds and the monsters started to become feral. {nl}The demons started to attack.{nl}It was like an incurable disease, we didn't see any sign of recovery.
+QUEST_20151001_002524	Still, the Goddesses were no where to be found.{nl}The Goddesses who we trusted disappeared without a trace and responded to nobody's prayers.{nl}People became mad and depressed, and soon only resignations were left behind.{nl}We started to prevent the chaos as much as we could, but as we did so, more misunderstandings were created. {nl}Death was always close-by and we gradually became dreary. {nl}
+QUEST_20151001_002525	We are becoming dreary to all the sadness and the pains.{nl}As if lifeless, we give up on everything and try not to do anything.{nl}It is like we are seeing the end of the history of the kingdom...{nl}We are just keep repeating the names of the Goddesses.{nl}
+QUEST_20151001_002526	I will end my writing here and leave the kingdom.{nl}I don't like my feeling of helplessness and the depressions.{nl}I am going to leave to a place where I can dream for hope again...
 QUEST_20151001_002527	Savior...{nl}Laima told us that she had dreamt about the collapse of the world.{nl}And among many Revelators, one Savior would save everyone.{nl}
-QUEST_20151001_002528	However, I cannot fully trust Laima's forecast.{nl}
+QUEST_20151001_002528	However, I cannot fully trust Laima's foresight.{nl}
 QUEST_20151001_002529	For the Savior that would appear some day...{nl}I should have been responsible for the flames of the world, but I neglected my duties.{nl}
```

Comment:
Laima can see the future. She isn't forecasting because to forecast means to >estimate< how something will be in the future. Instead, she has knowledge of the future, so she has foresight (ability to foresee or prepare wisely for the future) of the future.